requirements.txt

```diff
@@ -42,5 +42,6 @@ orjson>=3.9.15, <4
 mlrun-pipelines-kfp-common~=0.5.5
 mlrun-pipelines-kfp-v1-8~=0.5.4
 docstring_parser~=0.16
+timelength~=1.1
```

Comment:
bump to 3.x if you're at it

Reply:
I reverted the bump because they change the error handling and it requires some changes.
Moved the requirement back to server side so no effect on client
I liked the 1.x implementation better, perhaps worth upgrading to 2.x